src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/RecallNotificationServiceTest.kt

```diff
@@ -86,6 +88,54 @@ internal class RecallNotificationServiceTest {
     )
   }
 
+  @Test
+  fun `create recall notification uses latest revocation order if exists and creates recall summary and letter to probation`() {
```

Comment:
How about re-wording title as:
"create recall notification uses latest revocation order, or creates if not exists, and creates recall summary and letter to probation"
I think the titles need to make it clearer what each test is verifying please.